docs/src/documentation/03-components/03-information/toast/03-accessibility.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/toast/accessibility/
+---
+
+## Accessibility
+
+The Toast component has been designed with accessibility in mind, providing a non-intrusive way to display temporary notifications to users. It follows accessibility best practices to ensure all users, including those using assistive technologies, can perceive and understand these notifications.
+
+### Accessibility props
+
+| Name      | Type                               | Default    | Description                                                                                             |
+| :-------- | :--------------------------------- | :--------- | :------------------------------------------------------------------------------------------------------ |
+| role      | `"status" \| "alert"`              | `"status"` | The ARIA role for the toast. Use `"alert"` for important notifications requiring immediate attention    |
+| aria-live | `"polite" \| "assertive" \| "off"` | `"polite"` | Controls how urgently screen readers announce the content. Use `"assertive"` only for critical messages |
```

Comment:
Even after reading the examples it is not clear to me as a developer what's the difference in these and when should I use each